docs/contributing/install_dev.rst

```diff
@@ -216,3 +216,62 @@ repository root and use one of
 Either one of these commands will create a soft link to your cloned
 repository.  Any changes in Python code you make there will be there
 when you ``import plasmapy`` from an interactive session.
+
+Installing pre-commit
+=====================
+
+PlasmaPy uses the |pre-commit|_ framework to perform validations and
+automatically apply a consistent style to code contributions. Using
+|pre-commit|_ helps us find errors and shortens code reviews. PlasmaPy's
+pre-commit suite includes hooks such as:
+
+* ``check-ast`` to verify that the Python code is valid.
+* ``trailing-whitespace`` to remove trailing whitespace.
+* black_ to format code.
+* isort_ to sort imports.
+* nbqa_ to format notebooks.
```

Comment:
I guess we'd need https://github.com/nbQA-dev/nbQA in the common_links section for this to work!